src/console.rs

```diff
@@ -1,19 +1,7 @@
 //! Console function implementations for `.wasm`
 #![allow(dead_code)]
 
-use crate::{error::Explode, write_to_ipc};
-
 extern "C" {
 	/// Imported logging facade from JavaScript
 	fn console_log(is_string: bool, ipc_offset: usize);
```

Comment:
Since we are using the console_error_panic_hook crate, we don't need to import console.log from Javascript. I only used it for error handling